docs/modules/data-structures/pages/working-with-external-data.adoc

```diff
@@ -31,15 +31,88 @@ NOTE: The data store that you choose must be a centralized system that is access
 
 NOTE: The `MapStore` interface extends the `MapLoader` interface. Therefore, all methods and configuration parameters of the `MapLoader` interface are also available on the `MapStore` interface.
 
+== Defining External Data Stores (Beta)
```

Comment:
I think this content deserves its own page at the top level of *Configure and Manage Clusters*. It's defining a configuration procedure that connects the cluster to an external data store. You can then use this connection to interact with the data store in SQL, the Java Pipeline API, and MapStores. It doesn't make sense to hide this information on a page about MapStores.

An example of a configuration page is here: https://docs.hazelcast.com/hazelcast/latest/cp-subsystem/configuration This is a template that we should use. The only required sections are the description (what is this, why would you configure it, and how do you do it) and the list of config options in the table format: https://docs.hazelcast.com/hazelcast/latest/cp-subsystem/configuration#global-configuration-options

Reply:
I've moved the page: https://github.com/hazelcast/hz-docs/pull/514/commits/7733ba9a3fa3f42e9d7ff7d756f22d9f73880b87